### PR TITLE
Update doc for history and change section

### PR DESCRIPTION
**Pull request by minhtuev**
Updated Changes & History with MM 4.3

### CODE REVIEW

File: source/versions/changes.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@ Recent Changes
 ==============
 
 
+MindMeld 4.3
+-------------
+
+
+MindMeld 4.3 provides new features for developers to develop and streamline their conversational applications and incorporate the latest innovations in NLP. To read more about the latest changes in MindMeld 4.3, check out `our article on Medium <https://medium.com/p/9fededbab58e/>`_.
```
Comment (ktick) [inline source/versions/changes.rst:9]: We'll have to shelve the changes to this file till the Medium post is actually published.

File: source/versions/history.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
 Package History
 ===============
 
+4.3.1 (2020-06-17)
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Custom Action provides the ability for external applications to integrate custom dialogue management logic with MindMeld applications
```
Comment (vijay120) [inline source/versions/history.rst:10]:
```suggestion
- Custom Actions provide the ability for applications to integrate external dialogue management logic with MindMeld applications
```

File: source/versions/history.rst [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+
+- Question Answerer can leverage semantic embeddings (Elasticsearch 7 is required)
```
Comment (vijay120) [inline source/versions/history.rst:12]:
```suggestion
- Question Answerer can now leverage deep-learning based semantic embeddings to produce more relevant answers to queries
```

Comment (vijay120) [reply source/versions/history.rst:12]: I dont think we need to specify ES 7 here.

File: source/versions/history.rst [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+
+- Automatic slot filing provides a useful functionality for automatically prompting the user for missing entities or slots required to fulfill an intent
+
+- A new banking blueprint for enterprise use-cases
+
+- WhatsApp Integration
```
Comment (vijay120) [inline source/versions/history.rst:18]:
```suggestion
- WhatsApp Bot Integration with MindMeld
```

File: source/versions/history.rst [hunk 5]
```diff
@@ -1,6 +1,24 @@
...
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Custom Action provides the ability for external applications to integrate custom dialogue management logic with MindMeld applications
+
+- Question Answerer can leverage semantic embeddings (Elasticsearch 7 is required)
+
+- Automatic slot filing provides a useful functionality for automatically prompting the user for missing entities or slots required to fulfill an intent
```
Comment (vijay120) [inline source/versions/history.rst:14]:
```suggestion
- Automatic slot filing allows an intuitive way for developers to automatically prompt users for missing slots to fulfill an intent
```

File: source/versions/changes.rst (hunk 1 above)
Comment (vijay120) [inline source/versions/changes.rst:9]:
```suggestion
MindMeld 4.3 provides developers tools to build powerful question/answering systems, integrate with external clients, a new MindMeld enterprise blueprint and automatic slot filling. To read more about the latest changes in MindMeld 4.3, check out `our article on Medium <https://medium.com/p/9fededbab58e/>`_.
```

Comment (vijay120) [reply source/versions/changes.rst:9]: We could just incorp the quick summary I suggested below and not provide the external docs link yet.

**Comment by vijay120** [comment]
We should also mention "5. Configuring language and locale for your application" in the history section.

File: source/versions/history.rst (hunk 3 above)
Comment (minhtuev) [reply source/versions/history.rst:12]: It is specific to ES7+ right?

File: source/versions/history.rst [hunk 6]
```diff
@@ -1,6 +1,37 @@
...
+
+- Docker setup update to Elasticsearch 7
+
+- MindMeld application can configure language and locale setting in the application config file
+
+
+Bug fixes
+^^^^^^^^^
+
+- Addressed an issue which caused MindMeld to be unable to detect system entities without any context (for example: "december 21st")
```
Comment (vijay120) [inline source/versions/history.rst:28]:
```suggestion
- Addressed an issue which caused MindMeld to not detect system entities with no surrounding context (for example: "december 21st")
```

File: source/versions/history.rst [hunk 7]
```diff
@@ -1,6 +1,37 @@
...
+
+- Previously, MindMeld applications required Elasticsearch even if the application did not need it (i.e having no entity to resolve); read :doc:`Managing Dependencies <../userguide/optional_dependency>` for more details
```
Comment (vijay120) [inline source/versions/history.rst:30]:
```suggestion
- Previously, MindMeld applications called Elasticsearch even if the application did not functionally use it (i.e have no entity to resolve); This has been fixed using read :doc:`Managing Dependencies <../userguide/optional_dependency>`.
```

File: source/versions/history.rst [hunk 8]
```diff
@@ -1,6 +1,37 @@
...
+
+- Automatic slot filing allows an intuitive way for developers to automatically prompt users for missing slots to fulfill an intent
+
+- A new banking blueprint for enterprise use-cases
+
+- WhatsApp Bot Integration with MindMeld
+
+- Docker setup update to Elasticsearch 7
+
+- MindMeld application can configure language and locale setting in the application config file
```
Comment (vijay120) [inline source/versions/history.rst:22]:
```suggestion
- MindMeld application can configure language and locale settings in the application config file
```

**Pull request merged**